pkg/alerts/lcid_expiration.go

```diff
@@ -96,8 +97,8 @@ func checkForLCIDExpiration(
 
 	for _, currIntake := range result {
 
-		// Skip intake if it doesn't have an LCID
-		if currIntake.LifecycleExpiresAt == nil {
+		// Skip intake if it doesn't have an LCID or if it has a status of "NO GOVERNANCE"
+		if currIntake.LifecycleExpiresAt == nil || currIntake.ProcessStatus == null.StringFrom(string(models.SystemIntakeStatusNOGOVERNANCE)) {
```

Comment:
What was used to make the determination to only sip alerts on a status of `SystemIntakeStatusNOGOVERNANCE`?

I would imagine there'd be other statuses we'd want to skip from this list, but it's possible that it'd just be this one! (Just looking to confirm)
 https://github.com/CMSgov/easi-app/blob/8ce6c690c045154c930bd710035accfa386c68d0/pkg/models/system_intake.go#L21-L58

Reply:
Chatted on Slack - we'll go with this for now. If we need more statuses, we can add them later!